src/payment/strategies/digitalriver/digitalriver-payment-strategy.ts

```diff
@@ -113,18 +115,18 @@ export default class DigitalRiverPaymentStrategy implements PaymentStrategy {
                         bigpay_token: {
                             token: paymentData.instrumentId,
                         },
-                        verification_value: '000',
```

Comment:
I added it in the past because Bigpay had been given me an error that it required that value, but now seems like isn't required when you send a bigpay token.

Reply:
It is good to provide these as context either in the PR body or as a comment in the code when creating a PR for review. It reduces the review loop and makes it easier for the reviewer.